parsec/core/mountpoint/manager.py

```diff
@@ -43,28 +49,71 @@ def get_mountpoint_runner():
             return fuse_mountpoint_runner
 
 
-def get_default_mountpoint(device_id: DeviceID):
-    if os.name == "nt":
-        return Path("Z:")
-    else:
-        return Path(f"/media/{device_id}")
+async def disabled_runner(*args, **kwargs):
+    raise MountpointDisabled()
+
+
+class MountpointManager:
+    def __init__(self, base_mountpoint, fs, runner, nursery):
+        self._base_mountpoint = base_mountpoint
+        self._fs = fs
+        self._runner = runner
+        self._nursery = nursery
+        self._mountpoint_tasks = {}
+
+    async def mount_workspace(self, workspace: str):
+        if workspace in self._mountpoint_tasks:
+            raise MountpointAlreadyMounted(f"Workspace `{workspace}` already mounted.")
+
+        try:
+            await self._fs.stat(f"/{workspace}")
+        except FileNotFoundError as exc:
+            raise MountpointConfigurationError(f"Workspace `{workspace}` doesn't exist") from exc
+
+        mountpoint = self._base_mountpoint / f"{self._fs.device.user_id}-{workspace}"
```

Comment:
Why not use `self._base_mountpoint / f"{self._fs.device.user_id}/{workspace}"` instead?

Reply:
Good point !
My thinking was to have a clear naming in for the file explorer (nautilus detect the mountpoint and expose it with the name of the folder)

Reply:
But there is typically only one parsec client running at a time on a given machine right? So the `user_id` doesn't seem to provide much information :thinking: